custom_components/tuya_local/devices/blitzwolf_bwsh5_humidifier.yaml

```diff
@@ -84,17 +82,30 @@ secondary_entities:
             icon: "mdi:led-on"
           - dps_val: false
             icon: "mdi:led-off"
-  - entity: sensor
-    deprecated: humidifier
-    name: Current humidity
-    class: humidity
-    category: diagnostic
+  - entity: select
```

Comment:
Normally I would represent spray level as a fan entity, as I think the UI is a bit better, and it also comes with richer voice control.  It means duplicating the switch between both fan and humidifier, as both require a switch, and converting the levels to percentages rather than strings.

Reply:
I think it looks very ugly and not user friendly. Thera are 7 levels, so step between levels is 14.28571428571429, and converted to fan is 14.28%, 28.57% etc. How will user set level to 3 via voice control that way? I know it will step to nearest value, but I doubt that anyone will even try to set it that way...

Reply:
The levels do not need to be perfectly evenly spaced.  You can set them at 15%, 30%, 40%, ...

Reply:
No matter levels, this still looks ugly